### PR TITLE
Unify content routes

### PR DESCRIPTION
**Pull request by revolunet**
- unique route for content : `/contenu/[SOURCE]/[SLUG]`
 - less boilerplate for content pages

### CODE REVIEW

**Comment by revolunet** [comment]
attente PR précédentes pour rebase

File: packages/code-du-travail-frontend/src/content/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import * as Faq from "./Faq";
+import * as CodeDuTravail from "./CodeDuTravail";
+import * as Kali from "./Kali";
+import * as FicheServicePublic from "./FicheServicePublic";
+import * as FicheMinistere from "./FicheMinistere";
+import * as ModeleDeCourrier from "./ModeleDeCourrier";
+
+// source -> component mapping
+export default {
+  faq: Faq,
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/content/index.js:11]: why not call it `question` ? so we can get rid of the urlByTosourceType ?

Comment (revolunet) [reply packages/code-du-travail-frontend/src/content/index.js:11]: indeed!

**Comment by lionelB** [comment]
@revolunet need some rebase love !

**Pull request closed without merging**